wrapanapi/containers/providers/rhopenshift.py

```diff
@@ -35,6 +35,7 @@
     r"cfme-(\d)(\d)(\d)-(\d)-",     # cfme-520-1-   -> 5.2.0.1
     # 5 digits  (not very intelligent but no better solution so far)
     r"cfme-(?:nightly-)?(\d)(\d)(\d)(\d{2})-",   # cfme-53111-   -> 5.3.1.11, cfme-53101 -> 5.3.1.1
+    r"s-\w+-downstream-(\d)(\d)z-",  # s-tpl-downstream-57z-161116-2yunexwu
 ]
 VERSION_REGEXPS = map(re.compile, VERSION_REGEXPS)
```

Comment:
unrelated note please turn this one to a list, else we will be badly surprised on pyython3
can be a followup